msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1213,6 +1214,10 @@ public static String getSdkVersion() {
         return BuildConfig.VERSION_NAME;
     }
 
+    public static void showMsalRedirectUriInfo(Activity activity){
```

Comment:
Needs javadoc.....Also should we rename this to something like `showExpectedMsalRedirectUriInfo` (to indicate that the redirect uri displayed here isn't the one they have already registered (in case they have) but rather the one that we actually expect)?

Reply:
I like that suggestion.  Thanks